subsys/net/openthread/Kconfig.defconfig

```diff
@@ -269,6 +269,10 @@ config OPENTHREAD_CSL_TIMEOUT
 	int
 	default 20
 
+config OPENTHREAD_CSL_RECEIVER_LOCAL_TIME_SYNC
+	bool
+	default y if NRF_802154_SER_HOST
```

Comment:
for `OPENTHREAD_LIBRARY_AVAILABLE` we use depends on `SOC_NRF52840`  config. Should we use it here as well to keep consistency?